game/hud/src/widgets/Social/components/OrderContent.tsx

```diff
@@ -99,10 +100,12 @@ export class OrderContent extends React.Component<OrderContentProps, OrderConten
                       group={this.props.order}
                       userPermissions={this.props.order.myMemberInfo.permissions}
                       refetch={this.props.refetch} />
-      case 'admin': 
+      case 'admin':
         return <OrderAdmin dispatch={this.props.dispatch}
                            order={this.props.order}
                            refetch={this.props.refetch} />
+      case 'list':
```

Comment:
It's a separate switch, but the browse should be in the first switch anyway.  Will fix.